lib/jxl/enc_color_management.cc

```diff
@@ -849,9 +849,9 @@ bool ApplyCICP(const uint8_t color_primaries,
 
   const auto primaries = static_cast<Primaries>(color_primaries);
   const auto tf = static_cast<TransferFunction>(transfer_characteristics);
-  if (!EnumValid(tf) || tf == TransferFunction::kUnknown) return false;
-  if (!(EnumValid(primaries) || color_primaries == 12) ||
-      primaries == Primaries::kCustom) {
+  if (tf == TransferFunction::kUnknown || !EnumValid(tf)) return false;
```

Comment:
Do we need the swapping for `kUnknown` and `kCustom`? Are they not considered valid by `EnumValid`?

Reply:
No, they are considered valid, I just thought it would be nice to switch those statements around as well, in order to make the coder easier to read.